### PR TITLE
Add script directory for various testing tasks

### PR DESCRIPTION
**Pull request by ohohoreilly**
Adding script directory for various testing tasks
Starting to address the issues in: https://github.com/GoogleCloudPlatform/kubernetes/issues/7122

### CODE REVIEW

**Comment by lavalamp** [comment]
Please add .sh file extensions and update/add documentation

**Comment by preillyme** [comment]
@lavalamp: I've made the changes that you requested in https://github.com/preillyme/kubernetes/commit/e475ff60eb8a616e5161ef9406812c1e3d5137f3

File: www/script/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+## Running integration tests
+
+* `script/test-unit.sh` - run unit tests
+* `script/test-integration.sh` - run integration specs
+
+### Running tests headlessly
+
+Start Xvfb and export DISPLAY variable:
```
Comment (lavalamp) [inline www/script/README.md]: What is xvfb? Short description and/or link appreciated.

**Comment by lavalamp** [comment]
Please add boilerplate, too. Then LGTM.

**Comment by lavalamp** [comment]
Sorry for delay-- have been super busy.

**Comment by kprobst** [comment]
@preillyme Could you address @lavalamp 's comment and let me know when this PR is ready for re-review?

**Comment by bcbroussard** [comment]
@kprobst this is superseded by #8639 and #8659 and can be closed.

**Pull request closed without merging**